java/server/src/org/openqa/grid/images/displayhelpservlet.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="/assets/displayhelpservlet.css" media="all"/>
+  <script src="/assets/jquery-1.6.1.min.js" type="text/javascript"></script>
+  <script src="/assets/displayhelpservlet.js" type="text/javascript"></script>
+  <script type="text/javascript">
+    var json = Object.freeze('%s');
```

Comment:
since I don't like html inter-woven with java, what happens here is this html file acts as the template. the servlet loads this template, replaces `%s` with servlet config (via a string.format call), then javascript applies the json values onto the html... in this case the json will contain Se `version`, `type`, and the `console` link (which is different for hub, nodes, and standalones).